src/index.test.js

```diff
@@ -0,0 +1,74 @@
+import { getCertStoreRecords, parseDnsResults } from ".";
+
+const sampleDnsTextRecord = {
+  type: "openatts",
+  net: "ethereum",
+  netId: "3",
+  address: "0x0c9d5E6C766030cc6f0f49951D275Ad0701F81EC"
+};
+
+describe("getCertStoreRecords", () => {
+  const sampleDnsTextRecordWithDnssec = {
+    type: "openatts",
+    net: "ethereum",
+    netId: "3",
+    dnssec: true,
+    address: "0x0c9d5E6C766030cc6f0f49951D275Ad0701F81EC"
+  };
+  test("it should work", async () => {
+    expect(await getCertStoreRecords("ruijiechow.com")).toStrictEqual([sampleDnsTextRecordWithDnssec]);
+  });
+
+  test("it should return an empty array if there is no openatt record", async () => {
+    expect(await getCertStoreRecords("google.com")).toStrictEqual([]);
+  });
+
+  test("it should return an empty array with a non-existent domain", async () => {
+    expect(await getCertStoreRecords("thisdoesnotexist.gov.sg")).toStrictEqual([]);
+  });
+});
+
+describe("parseDnsResults", () => {
+  test("it should work", () => {
+    const sampleRecord = [
+      {
+        name: "ruijiechow.com.",
+        type: 16,
+        TTL: 110,
+        data: '"openatts net=ethereum netId=3 address=0x0c9d5E6C766030cc6f0f49951D275Ad0701F81EC"'
```

Comment:
quotes is just a quirk of dns txt record i think
as for prepending with OA, i don't feel strongly about sticking with openatts but my choice was guided by two constraints, 

1) 255 char limit
2) not having it be so short that it might end up colliding with some other random dns txt record

in view of that, what do you think?

Reply:
Just felt that we dont use the term "openatts" anywhere else.